specs/agents/configuration.md

```diff
@@ -69,3 +69,33 @@ Failure to process one config attribute should not affect processing of others.
 #### Feature flag
 
 Agents should implement a [configuration option](https://docs.google.com/spreadsheets/d/1JJjZotapacA3FkHc2sv_0wiChILi3uKnkwLTjtBmxwU), (`CENTRAL_CONFIG`) which lets users disable the central configuration polling.
+
+
+### Configuration options
+
+This section specifies *some* configuration options common to two or more APM
+agents. Some configuration options are discussed elsewhere in this repository
+if they better match a particular section -- for example
+[`transaction_ignore_urls`](./tracing-instrumentation-http.md#transaction_ignore_urls-configuration).
+
+#### `ELASTIC_APM_DISABLE_SEND`
```

Comment:
Okay, I'll change it. 

### off-topic for this PR

Perhaps this discussion has already been had, but I'll state my case.
I was `waffling_on` `WHICH_STYLE` to `ELASTIC_APM_USE` here. The options being:
1. `UPPER_NO_PREFIX` -- used by the [comparison chart](https://docs.google.com/spreadsheets/d/1JJjZotapacA3FkHc2sv_0wiChILi3uKnkwLTjtBmxwU/edit#gid=0) for brevity
2. `ELASTIC_APM_UPPER_WITH_PREFIX` -- All agents support the environment variables and include these strings in their configuration docs.
3. `lower_snake_case` -- Used by Java, Python, Ruby, and PHP. These strings do *not* in general appear in docs for .NET, Node.js, and Go.
4. `camelCase` -- Only used by Node.js, so obviously isn't a candidate for shared docs.

I'm a fan of [the grep test](http://jamie-wong.com/2013/07/12/grep-test/) for allowing one to find relevant docs and code for a thing. `ELASTIC_APM_UPPER_WITH_PREFIX` is the only style that will reliably find hits in user docs, without `-i` for case-insensitive and with `-w` for whole word match to avoid false positives for common config works like "environment", "enabled", "active", "instrument", "debug", "hostname".  Also being able to grep for `ELASTIC_APM_` sure is nice to find "all configurable things mentioned in the docs".

Anyway, it is not a big thing. `lower_snake_case` is fine.

- I think it would be worth adding a note to the top of configuration.md about preferring to use the `server_url` style for docs. There are a number of uses of `ELASTIC_APM_FOO` in the current docs.
- Would it be reasonable (modulo the make-work) to lazily swith over the comparison chart to `that_style`?

Reply:
That said, generally communicating (in the docs, in chat and emails with the team) in terms of `server_url` is certainly much easier to type than `ELASTIC_APM_SERVER_URL`.